Handling CephClusterReadOnlyMode (#23)
red-hat-storage/ocs-sop: sop/modules/OSD/partials/ocs_extend_cluster.adoc

> To Extend the cluster , and reset is back, follow the below mentioned steps

Reviewer: What is `reset is back` here ?

Reviewer: Also we are not extending the cluster, we are extending the cluster full threshold

> *Extend the ceph cluster by 0.02*

Reviewer: ```suggestion
*Extend the ceph cluster by increasing the threshold to 87%*
```

> To extend the ceph cluster, you can now just run the following command 

Reviewer: ```suggestion
Execute the following command to extend the ceph cluster full threshold.
```

> ----

Reviewer: add `[source,role="execute"]` before this line

oc process -n openshift-storage ocs-extend-cluster | oc create -f -
----

> Now the ceph cluster threshold is extended from 0.85 to 0.87, this will make the cluster to get out of HEALTH_ERR state.

Reviewer: ```suggestion
Now the ceph cluster threshold is extended from 0.85 to 0.87, this will make the cluster to get out of HEALTH_ERR state and Readonly mode.
```

> Now the ceph cluster is extended by 0.02, the cluster state would become healthy
> Notify the customer once the cluster is healthy. When the customer is done deleting data ,
> you need to return the threshold to the previous value.
>  
> but before you reset your cluster back to the previous threshold value, you should delete the existing job resource:

Reviewer: ```suggestion
Now the ceph cluster threshold is extended from 0.85 to 0.87, this will make the cluster to get out of HEALTH_ERR state.
Notify the customer once it is out of HEALTH_ERR state to delete some of the existing data. When the customer is done deleting data ,
you need to revert the threshold to the previous value.
 
Before reverting the threshold, the existing job resource should be deleted:
```

> ----

Reviewer: add [source,role="execute"] before this line

oc delete job ocs-extend-cluster-job -n openshift-project
----

> Once you have deleted the old job, you can re-run the extend cluster job again to reset it back to the previous threshold value

Reviewer: ```suggestion
Once the old job is removed, re-run the extend cluster job to revert it to the previous threshold value
```

> ----

Reviewer: add [source,role="execute"] before this line
